mysql/resource_grant.go

```diff
@@ -183,7 +183,7 @@ func (t *ProcedurePrivilegeGrant) GetPrivileges() []string {
 }
 
 func (t *ProcedurePrivilegeGrant) SQLGrantStatement() string {
-	stmtSql := fmt.Sprintf("GRANT %s ON %s %s.%s TO %s", strings.Join(t.Privileges, ", "), t.ObjectT, t.GetDatabase(), t.CallableName, t.UserOrRole.SQLString())
+	stmtSql := fmt.Sprintf("GRANT %s ON %s %s.`%s` TO %s", strings.Join(t.Privileges, ", "), t.ObjectT, t.GetDatabase(), t.CallableName, t.UserOrRole.SQLString())
```

Comment:
This looks terrible. Could you please introduce a getter to `t.CallableName` similar to `t.GetDatabase()`?

This looks like something is not quoted or maybe quoted twice because of inconsistency between those two.

Reply:
@petoju fixed!